### PR TITLE
fix: size

### PR DESCRIPTION
**Pull request by rbalet**
## Description
This pull request contain the following update, _in that order_.

### 1: Core - update: dependencies
a. I've updated every dependencies, getting from `v14.2.1` to `v16.2.0`
b. I've bumped the library version to `v16.1.0`

### 2: Optimize - module : into standalone
Now that we have `v16.2.0`, I'v migrate the component into a standalone one and remove the unused `CommonModule` dependencies. 

This will make the package even smaller 🤩 

### 3: optimize - remove/unused : PLATFORM_ID
I saw that this token wasn't used. so I removed it.

`constructor(@Inject(PLATFORM_ID) private platformId: Object) {}`

### 4: Fix - size: on div element
This piece of code couldn't work because size would be equal to `200` while style.width expect to receive `200px`. 
```typescript
this.div.nativeElement.style.width = +this.size;
this.div.nativeElement.style.height = +this.size;
```
into
```typescript
this.div.nativeElement.style.width = this.size.toString() + "px";
this.div.nativeElement.style.height = this.size.toString() + "px";
```

_I also added a type to the `@ViewChild` ^^.  
->  `@ViewChild("elem", { static: true }) div!: ElementRef<HTMLElement>;`

----

_I made three different commit, so that we can easily revert one, if not wanted, but I do think they are all quite useful_

### CODE REVIEW

**Comment by werthdavid** [comment]
Much appreciated thank you!

**Comment by werthdavid** [comment]
Does that work without the module? Importing the component directly?

File: projects/ngx-kjua/package.json [hunk 1]
```diff
@@ -14,6 +14,10 @@
       "email": "werth.david@gmail.com"
     }
   ],
+  "publishConfig": {
```
[the remark this answers is not in the document]
Comment (rbalet) [reply projects/ngx-kjua/package.json]: @werthdavid Damn, was for my internal testing (Just to be sure I didn't break your code I do push on a private server first '^^), same for the `@megaphone/ngx-kjua`. 

Sorry for that, removed in [this commit](https://github.com/werthdavid/ngx-kjua/pull/80/commits/a824abb72c726eecb475473961c32d28158d6b4f)

**Comment by rbalet** [comment]
> Does that work without the module? Importing the component directly?

Yes it does, this is the new way of doing for angular 16. [Read more](https://angular.io/guide/standalone-components).

It is really useful for smaller project that doesn't need to import a tons of module. 

It does also generate a smaller bundle size ;)

**Review by werthdavid — APPROVED** [review]

**Comment by rbalet** [comment]
@werthdavid It will create an error on existing project though, since the people will have to import the Component instead of the module.

I've updating the Readme real quick

**Pull request merged**